src/pages/HomePage/HomePage.js

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
-import { Heading1, Heading2 } from './HomePage.style';
+// import { Heading1, Heading2 } from './HomePage.style';
+import Header from '../../components/Header/Header';
+import Logo from '../../assets/images/logo.svg';
+
+const headerLinks = [
```

Comment:
Since the header is the same on both pages you could extract it somewhere else so it can be reused between the pages